ui/tests/acceptance/pki/pki-action-forms-test.js

```diff
@@ -0,0 +1,254 @@
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+import authPage from 'vault/tests/pages/auth';
+import logout from 'vault/tests/pages/logout';
+import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
+import { runCommands } from 'vault/tests/helpers/pki/pki-run-commands';
+import { SELECTORS as S } from 'vault/tests/helpers/pki/workflow';
+import { issuerPemBundle } from 'vault/tests/helpers/pki/values';
+
+module('Acceptance | pki action forms test', function (hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(async function () {
+    await authPage.login();
+    // Setup PKI engine
+    const mountPath = `pki-workflow-${new Date().getTime()}`;
+    await enablePage.enable('pki', mountPath);
+    this.mountPath = mountPath;
+    await logout.visit();
+  });
+
+  hooks.afterEach(async function () {
+    await logout.visit();
+    await authPage.login();
+    // Cleanup engine
+    await runCommands([`delete sys/mounts/${this.mountPath}`]);
+    await logout.visit();
+  });
+
+  module('import', function (hooks) {
+    setupMirage(hooks);
+
+    hooks.beforeEach(function () {
+      this.pemBundle = issuerPemBundle;
+    });
+
+    test('happy path', async function (assert) {
+      await authPage.login(this.pkiAdminToken);
+      await visit(`/vault/secrets/${this.mountPath}/pki/overview`);
+      assert.strictEqual(currentURL(), `/vault/secrets/${this.mountPath}/pki/overview`);
+      await click(S.emptyStateLink);
+      assert.strictEqual(currentURL(), `/vault/secrets/${this.mountPath}/pki/configuration/create`);
+      assert.dom(S.configuration.title).hasText('Configure PKI');
+      assert.dom(S.configuration.emptyState).exists({ count: 1 }, 'Shows empty state by default');
+      await click(S.configuration.optionByKey('import'));
+      assert.dom(S.configuration.emptyState).doesNotExist();
+      // Submit before filling out form shows an error
+      await click('[data-test-pki-import-pem-bundle]');
+      assert.dom('[data-test-alert-banner="alert"]').hasText('Error please upload your PEM bundle');
+      // Fill in form data
+      await click('[data-test-text-toggle]');
+      await fillIn('[data-test-text-file-textarea]', this.pemBundle);
+      await click('[data-test-pki-import-pem-bundle]');
+
+      assert.strictEqual(
+        currentURL(),
+        `/vault/secrets/${this.mountPath}/pki/configuration/create`,
+        'stays on page on success'
+      );
+      assert.dom(S.configuration.title).hasText('View imported items');
+      assert.dom(S.configuration.importForm).doesNotExist('import form is hidden after save');
+      assert.dom(S.configuration.importMapping).exists('import mapping is shown after save');
+      await click('[data-test-done]');
+      assert.strictEqual(
+        currentURL(),
+        `/vault/secrets/${this.mountPath}/pki/overview`,
+        'redirects to overview when done'
+      );
+    });
+    skip('with many imports', async function (assert) {
```

Comment:
is this skipped to just get us moving? Maybe a TODO to return to this?

Reply:
Ah, I see a lot of skips on this. Just confirming there are tickets to return to these?